control-plane/csi-driver/src/bin/node/node.rs

```diff
@@ -368,23 +369,29 @@ impl node_server::Node for Node {
             ));
         }
         match sys::statfs::statfs(&*msg.volume_path) {
-            Ok(info) => Ok(Response::new(NodeGetVolumeStatsResponse {
-                usage: vec![
-                    csi::VolumeUsage {
-                        total: info.blocks() as i64 * info.block_size(),
-                        unit: csi::volume_usage::Unit::Bytes as i32,
-                        available: info.blocks_available() as i64 * info.block_size(),
-                        used: (info.blocks() - info.blocks_free()) as i64 * info.block_size(),
-                    },
-                    csi::VolumeUsage {
-                        total: info.files() as i64,
-                        unit: csi::volume_usage::Unit::Inodes as i32,
-                        available: info.files_free() as i64,
-                        used: (info.files() - info.files_free()) as i64,
-                    },
-                ],
-                volume_condition: None,
-            })),
+            Ok(info) => match info.filesystem_type() {
+                sys::statfs::EXT4_SUPER_MAGIC | FsType(XFS_MAGIC) => {
```

Comment:
Is the match in line. 72 not enough?

Reply:
It is, but this way is sort of a hack. Let's say if tomorrow MayaStor is supporting 10 other file systems, this part of code would need to be changed each time a new fs support is added. Hence I've plans to figure out what should be done here such that zeros are returned for volume capacity used.

Reply:
Also right now it lazily throws an error, but actually it might be better if it returns zero values for used capacity and the total capacity is retuned as is, I'll take a look at it sometime soon and target to ACK all issues in a week